### DISCUSSION TITLE
Receive FCM notifications.

### DISCUSSION THREAD

**Discussion by Autoneer** [SchneiderInfosystems/FB4D#137] [General]
Can you use FB4D to subscribe to a document to receive data change notifications, even if the app is closed? 
and if so do you have sample app or code for this ?

**Comment by SchneiderInfosystems** [comment]
I assume you ask this question for an App on the Android platform?

No, FB4D has no such capabilities because there is no running background service that can wake up your app. Of course, you could implement such a service by yourself, but this is probably not an easy way.
The only function to wake up a closed app on Android with Firemonkey is to use the Push Notification via Firebase SDK on Android. For more, see this:
https://blogs.embarcadero.com/firebase-android-push-notification-support-with-rad-studio-10-3-1
